finality-aleph/src/sync/service.rs

```diff
@@ -293,7 +294,22 @@ where
         );
         match self.handler.handle_request(request) {
             Ok(Action::Response(response_items)) => {
-                self.send_to(NetworkData::RequestResponse(response_items), peer);
+                let mut limiter = MsgLimiter::new(&response_items);
```

Comment:
What do you think about following the pattern of `VersionedWrapper` and making it implementation detail of the `Network` implementation?

Reply:
Im not sure what you exactly mean here

Reply:
not done, as discussed offline :P